doc/changelogs/CHANGELOG_V17.md

```diff
@@ -45,6 +45,30 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="17.9.1"></a>
+
+## 2022-05-31, Version 17.9.1 (Current), @ruyadorno
```

Comment:
Is it Current or Maintenance?

Reply:
I originally named it `(Maintenance)` but changed it to `(Current)`, ref: https://github.com/nodejs/node/pull/43256#discussion_r885548640